skops/io/tests/test_persist.py

```diff
@@ -804,3 +804,30 @@ def test_loads_from_str():
     msg = "Can't load skops format from string, pass bytes"
     with pytest.raises(TypeError, match=msg):
         loads("this is a string")
+
+
+class _BoundMethodHolder:
+    """Used to test the ability to serialise and deserialize bound methods"""
```

Comment:
```suggestion
    """Used to test the ability to serialize and deserialize bound methods"""
```

Either British or American but not both ;) It seems we went with American in this repo.